prow/config.yaml

```diff
@@ -7180,6 +7180,28 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  - name: pull-tf-minigo-cc
+    agent: kubernetes
+    always_run: true
+    skip_report: true
+    #context: pull-tf-minigo-cc
+    branches:
+    - master
```

Comment:
@amj Should we run this for PRs to all branches or just the master branch?

Reply:
just PRs to master is fine.